broker/machinebroker/server/machine_networkinterface_attach.go

```diff
@@ -30,36 +30,45 @@ type IronCoreNetworkInterfaceConfig struct {
 	NetworkID  string
 	IPs        []commonv1alpha1.IP
 	Attributes map[string]string
-	Labels     map[string]string
```

Comment:
Can we keep the  `Labels     map[string]string` here? Maybe also add second field hier for the network labels.
It's only a internal representation  (not related to the API) so we can also remove it easily and avoid encoding and decoding.